packages/@sfx/search-plugin/src/search-plugin.ts

```diff
@@ -0,0 +1,59 @@
+import { Plugin, PluginRegistry, PluginMetadata } from '@sfx/core';
+import { BridgeConfig, BrowserBridge, Query } from 'groupby-api';
+
+/**
+ * This plugin exposes an instance of the GroupBy search client
+ */
+export default class SearchPlugin implements Plugin {
+  get metadata(): PluginMetadata {
+    return {
+      name: 'search',
+      depends: [],
+    };
+  }
+
+  /**
+   * The values that the Search Data Source plugin expose to the Core entity.
+   */
+  browserBridge: SearchPluginExposedValue;
+
+  /**
+   * Instantiates an instance of the search browserBridge plugin
+   *
+   * @param options The options to instantiate the search data source browser bridge client with.
+   */
+   constructor(options: SearchPluginOptions) {
+     const customerId = options.customerId;
+     const https = !options.https ? false : true;
```

Comment:
Are you trying to convert `options.https` to a boolean here? If so, you can just do:
```suggestion
     const https = !!options.https;
```